lib/config-generator.js

```diff
@@ -399,6 +399,17 @@ class ConfigGenerator {
             }));
         }
 
+        if (this.webpackConfig.useSvelte) {
+            rules.push(applyRuleConfigurationCallback('svelte', {
```

Comment:
Would require to update https://symfony.com/doc/current/frontend/encore/advanced-config.html#having-the-full-control-on-loaders-rules, thanks!

Reply:
yes after the merge :)